app/controllers/groups_controller.rb

```diff
@@ -10,27 +10,14 @@ class GroupsController < ApplicationController
 
   helper_method :team
 
-  # GET /teams/1/groups
-  def index
-    authorize team, :team_member?
-    @groups = team.groups
-    groups_breadcrumbs
-
-    teammembers = team.teammembers.list
-    @teammembers = teammembers.includes(:user).sort_by { |tm| tm.label.downcase }
-
-    respond_to do |format|
-      format.html # index.html.haml
-    end
-  end
-
   # GET /teams/1/groups/1
   def show
-    @group = team.groups.find(params[:id])
-    authorize @group
+    authorize team
```

Comment:
well, the result is the same in this case for the moment ... but we should not authorize on team here !